lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_mysql_SUITE.erl

```diff
@@ -0,0 +1,260 @@
+    end.
+
+mysql_config(MysqlHost, MysqlPort0, GroupType, BridgeType) ->
+    MysqlPort = integer_to_list(MysqlPort0),
+    Server = MysqlHost ++ ":" ++ MysqlPort,
+    Name = iolist_to_binary(io_lib:format("~s-~s", [?MODULE, GroupType])),
+    SslEnabled =
+        case GroupType of
+            tcp -> "false";
+            tls -> "true"
+        end,
+    ConfigString =
+        io_lib:format(
+            "bridges.~s.~s {\n"
+            "  enable = true\n"
+            "  server = ~p\n"
+            "  database = ~p\n"
+            "  username = ~p\n"
+            "  password = ~p\n"
+            "  sql = ~p\n"
+            "  ssl = {\n"
+            "                 enable = ~w\n"
+            "  }\n"
+            "}",
+            [
+                BridgeType,
+                Name,
+                Server,
+                ?MYSQL_DATABASE,
+                ?MYSQL_USERNAME,
+                ?MYSQL_PASSWORD,
+                ?SQL_BRIDGE,
+                SslEnabled
+            ]
+        ),
+    {Name, parse_and_check(ConfigString, BridgeType, Name)}.
+
+parse_and_check(ConfigString, BridgeType, Name) ->
+    {ok, RawConf} = hocon:binary(ConfigString, #{format => map}),
+    hocon_tconf:check_plain(emqx_bridge_schema, RawConf, #{required => false, atom_key => false}),
+    #{<<"bridges">> := #{BridgeType := #{Name := Config}}} = RawConf,
+    Config.
+
+create_bridge(Config) ->
+    BridgeType = ?config(mysql_bridge_type, Config),
+    Name = ?config(mysql_name, Config),
+    MysqlConfig = ?config(mysql_config, Config),
+    emqx_bridge:create(BridgeType, Name, MysqlConfig).
+
+delete_bridge(Config) ->
+    BridgeType = ?config(mysql_bridge_type, Config),
+    Name = ?config(mysql_name, Config),
+    emqx_bridge:remove(BridgeType, Name).
+
+create_bridge_http(Params) ->
+    Path = emqx_mgmt_api_test_util:api_path(["bridges"]),
+    AuthHeader = emqx_mgmt_api_test_util:auth_header_(),
+    case emqx_mgmt_api_test_util:request_api(post, Path, "", AuthHeader, Params) of
+        {ok, Res} -> {ok, emqx_json:decode(Res, [return_maps])};
+        Error -> Error
+    end.
+
+query(Config, SqlQuery) ->
+    BridgeType = ?config(mysql_bridge_type, Config),
+    Name = ?config(mysql_name, Config),
+    ResourceID = emqx_bridge_resource:resource_id(BridgeType, Name),
+    ?assertMatch({ok, connected}, emqx_resource:health_check(ResourceID)),
+    emqx_resource:simple_sync_query(ResourceID, {sql, SqlQuery}).
+
+send_message(Config, Payload) ->
+    Name = ?config(mysql_name, Config),
+    BridgeType = ?config(mysql_bridge_type, Config),
+    ResourceID = emqx_bridge_resource:resource_id(BridgeType, Name),
+    ?assertMatch({ok, connected}, emqx_resource:health_check(ResourceID)),
+    % TODO: Check why we can't use send_message directly!
+    % BridgeID = emqx_bridge_resource:bridge_id(Type, Name),
+    % emqx_bridge:send_message(BridgeID, Payload).
+    emqx_resource:simple_sync_query(ResourceID, {send_message, Payload}).
```

Comment:
Whenever I try to use `emqx_bridge:send_message/2` in the test, an error report is generated with `{error, badarg}` that stems from ecpool/gproc somewhere being called from `emqx_connector_mysql:on_sql_query/6`. I need help to figure out why this happens. The test works fine when using `emqx_resource:simple_sync_query/2`.

Reply:
Testing locally with [this uncommented](https://github.com/olcai/emqx/blob/580992b5069e9464a17f99e66ff5c748595af041/lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_mysql_SUITE.erl#L187-L188) worked as-is (except for `s/Type/BridgeType`) for me, without those errors. :thinking: 

I had to insert a couple sleeps between the write and read in the tests, probably because it's writing asynchronously.